README.md

```diff
@@ -81,15 +81,17 @@ pip install -e homu
 
 1. Copy `cfg.sample.toml` to `cfg.toml`, and edit it accordingly.
 
-2. Create a GitHub account that will be used by Homu. You can also use the
-   existing account. In the account settings, register a new application and
-   generate a new access token.
+2. Create a GitHub account that will be used by Homu. You can also use an
+   existing account. In the [account settings][settings], register a new
+   application and generate a new access token (with the `repo` permission).
+
 
 3. Add a Webhook to your repository:
 
  - Payload URL: `http://HOST:PORT/github`
  - Content type: `application/json`
  - Secret: The same as `repo.NAME.github.secret` in cfg.toml
+ - Events: `push`, `issue_comment`.
```

Comment:
Sorry, I forgot an event! `pull_request` is also needed. However, as the GitHub UI specifies those events as human-friendly names, now I think we'd better use `Issue comment`, `Pull Request`, and `Push`.